src/main/java/com/teradata/tpcds/Session.java

```diff
@@ -47,17 +52,26 @@ public Session(double scale, String targetDirectory, String suffix, Optional<Tab
 
     public Session(double scale, String targetDirectory, String suffix, Optional<Table> table, String nullString, char separator, boolean doNotTerminate, boolean noSexism, int parallelism, int chunkNumber, boolean overwrite)
     {
+        this(scale, targetDirectory, suffix, table, nullString, separator, doNotTerminate, noSexism, parallelism,
+                chunkNumber, overwrite, null);
+    }
+
+    public Session(double scale, String targetDirectory, String suffix, Optional<Table> table, String nullString,
+                   char separator, boolean doNotTerminate, boolean noSexism, int parallelism, int chunkNumber,
+                   boolean overwrite, Configuration hadoopConfig)
+    {
         this.scaling = new Scaling(scale);
         this.targetDirectory = targetDirectory;
         this.suffix = suffix;
-        this.table = table;
+        this.table = table.isPresent() ? table.get() : null;
```

Comment:
while you're here, it would be great to add `requireNonNull()` checks for the fields that shouldn't be null.